Monika After Story/game/script-moods.rpy

```diff
@@ -882,3 +882,26 @@ label mas_mood_excited:
             m 3eua "I know I'm always excited to see you every day."
             m 1hub "Either way, I'm glad that you're happy!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_mood_database,
+            eventlabel="mas_mood_burnt",
+            prompt="...burnt out.",
+            category=[store.mas_moods.TYPE_BAD],
+            unlocked=True
+        ),
+        code="MOO"
+    )
+
+label mas_mood_burnt:
+            m 1wud "Oh!"
```

Comment:
```suggestion
    m 1wud "Oh!"
```
Hey there, Retro! Looks like your indentation is too far. Indentation is very important in RenPy - You seem to have used 8 spaces. MAS uses 4.